From e2127b9b932ba00811d5023c5ea69a12a857b244 Mon Sep 17 00:00:00 2001
From: Alex Murkoff <79400603+alexlnkp@users.noreply.github.com>
Date: Wed, 22 May 2024 16:24:04 +0700
Subject: [PATCH] bump CMake minimum required version to avoid warnings

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2d3ecfc7..21607e94 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.!
 
-cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
 file(STRINGS "VERSION.txt" SPM_VERSION)
 message(STATUS "VERSION: ${SPM_VERSION}")
 